arm-storage/2015-06-15/swagger/storage.json

```diff
@@ -1,17 +1,19 @@
 {
   "swagger": "2.0",
   "info": {
-    "title": "StorageManagementClient",
-    "description": "The Storage Management Client.",
-    "version": "2015-06-15"
+    "title": "StorageManagement",
+    "description": "The Azure Storage Management API.",
+    "version": "2015-06-15",
+    "x-ms-code-generation-settings": {
+      "name": "StorageManagementClient"
+    }
   },
   "host": "management.azure.com",
   "schemes": [
     "https"
   ],
   "consumes": [
-    "application/json",
-    "text/json"
```

Comment:
@bganapa Does your service for this (old) api-version not support text/json? If it doesn't, then removing it should be fine, if it does support it, removing it may cause a breaking change for someone generating code from the swagger file.